core/src/main/java/com/auth0/authentication/DelegationRequest.java

```diff
@@ -99,4 +100,14 @@
     public void start(final BaseCallback<T> callback) {
         request.start(callback);
     }
+
+    /**
+     * Executes the HTTP request against Auth0 API
```

Comment:
Let's void talking about generic HTTP request since its related to a Delegation request. 
> Might be useful to explain that the response is dependant on the app target.